src/utils.rs

```diff
@@ -143,6 +157,90 @@ pub fn to_human_size(input: U256) -> String {
     tmp.iter().rev().collect()
 }
 
+/// Creates and inserts a given number of empty blocks into the node, with a given interval between them.
+/// The blocks will be empty (contain no transactions).
+/// The test system contracts will be used to force overwriting the block number and timestamp in VM state,
+/// otherwise the VM will reject subsequent blocks as invalid.
+pub fn mine_empty_blocks<S: std::fmt::Debug + ForkSource>(
+    node: &mut InMemoryNodeInner<S>,
+    num_blocks: u64,
+    interval_ms: u64,
+) {
+    // build and insert new blocks
+    for _ in 0..num_blocks {
+        node.current_miniblock = node.current_miniblock.saturating_add(1);
+
+        let block = Block {
+            hash: compute_hash(node.current_miniblock as u32, H256::zero()),
+            number: node.current_batch.into(),
+            timestamp: node.current_timestamp.into(),
+            ..Default::default()
+        };
+
+        node.block_hashes.insert(node.current_miniblock, block.hash);
+        node.blocks.insert(block.hash, block);
+
+        // leave node state ready for next interaction
+        node.current_timestamp = node.current_timestamp.saturating_add(interval_ms);
+        node.current_batch = node.current_batch.saturating_add(1);
+    }
+
+    // roll the vm
+    let (keys, bytecodes) = {
+        let mut storage_view: StorageView<&ForkStorage<S>> = StorageView::new(&node.fork_storage);
+        let mut oracle_tools = OracleTools::new(&mut storage_view, HistoryEnabled);
+
+        let bootloader_code = &node.system_contracts.playground_contracts;
+
+        let block_context = BlockContext {
+            block_number: node.current_batch - 1,
+            block_timestamp: node.current_timestamp,
+            ..node.create_block_context()
+        };
+        let block_properties = InMemoryNodeInner::<S>::create_block_properties(bootloader_code);
+
+        // init vm
+        let mut vm = init_vm_inner(
+            &mut oracle_tools,
+            BlockContextMode::OverrideCurrent(block_context.into()),
+            &block_properties,
+            BLOCK_GAS_LIMIT,
+            bootloader_code,
+            TxExecutionMode::VerifyExecute,
+        );
+
+        vm.execute_till_block_end(BootloaderJobType::BlockPostprocessing);
```

Comment:
Similar to above comment, should this be `BootloaderJobType::TransactionExecution`?